helm/examples/local/README.md

```diff
@@ -27,6 +27,14 @@ Examples of use:
   helm install localfile examples/local/hpcc-localfile --set common.hostpath=/User/myuser/hpccdata
   helm install mycluster hpcc/ --set global.image.version=latest -f examples/local/values-localfile.yaml
 
+#### Docker desktop (using hyper-v):
+
+  With a host directory of c:\hpccdata
+  It should be added to Docker Desktop Settings->Resources->FILE SHARING
+
+  helm install localfile examples/local/hpcc-localfile --set common.hostpath=/c/hpccdata
+  helm install mycluster hpcc/ --set global.image.version=latest -f examples/local/values-localfile.yaml
+
```

Comment:
In addition, you must add the folder in Docker Desktop's settings for Resources>File Sharing.

Reply:
changed